codility_training/lessons.lesson02.Arrays.CyclicRotation/khw970421.js

```diff
@@ -0,0 +1,9 @@
+function solution(A, K) {
+    // write your code in JavaScript (Node.js 8.9.4)4
+    let b=[];
+    for(let i=0;i<A.length;i++)
```

Comment:
for 문 안에 A.length 를 체크를 하면 포문을  i 번만큼 순회하면서 A.length 를 매번 탐색하는데 시간이 소요 됩니다.
```suggestion
    const aLen = A.length;
    for(let i = 0; i < aLen; i++) {
```

Reply:
위에 준일이형한테 의견을 남겼는데 시간소요 차이군여 역시 확인했습니다